pki.spec

```diff
@@ -514,6 +514,13 @@ Conflicts:        ipa-server < 4.7.1
 Conflicts:        freeipa-server < 4.7.1
 %endif
 
+Provides:         bundled(js-backbone) = 1.4.0
+Provides:         bundled(js-bootstrap) = 3.4.1
+Provides:         bundled(js-jquery) = 3.5.1
```

Comment:
Only `js-jquery` exists in RPM land, so I made up names for the rest. That sound OK?

Reply:
Sounds good to me.

Separate issue, since we're vendoring anyway, should we set up a new folder for all vendored packages?

Reply:
Sure. Perhaps we make that a separate PR though.